fiat-constify/src/main.rs

```diff
@@ -59,29 +59,33 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
 /// Build a toplevel attribute with the given name and comma-separated values.
 fn build_attribute(name: &str, values: &[&str]) -> Attribute {
-    let span = Span::call_site();
     let values = values
         .iter()
         .map(|value| build_path(value))
         .collect::<Vec<_>>();
+    let path = build_path(name);
+    let tokens = quote! { (#(#values),*) };
```

Comment:
I spot checked the generated code and this appears to be adding an extra level of paren nesting.

Here's a diff of the generated code:

```
```
-#![allow(
-    dead_code,
-    rustdoc::bare_urls,
-    rustdoc::broken_intra_doc_links,
-    unused_assignments,
-    unused_mut,
-    unused_variables
-)]
+# ! [allow ((clippy :: identity_op , clippy :: unnecessary_cast , dead_code , rustdoc :: broken_intra_doc_links , unused_assignments , unused_mut , unused_variables))]
```
```